### PR TITLE
Hotfix/2015 07 08 1145 journal csv

### PR DESCRIPTION
**Pull request by Steven-Eardley**
issue #790

### CODE REVIEW

**Comment by Steven-Eardley** [comment]
hmm, the .gitignore is still appearing as binary.

**Comment by emanuil-tolev** [comment]
Weird, last commit shows it ok? Let me check locally too

**Comment by emanuil-tolev** [comment]
Yeah it works locally. I can see text diffs from before the 30 Mar and now I see a text diff of 2 removed lines again. I think it's ok now, probably next PR to change it will work in github.

File: portality/formcontext/xwalk.py [hunk 1]
```diff
@@ -917,7 +917,8 @@ def obj2form(cls, obj):
 
         forminfo['subject'] = []
         for s in bibjson.subjects():
-            forminfo['subject'].append(s['code'])
+            if "code" in s:
```
Comment (emanuil-tolev) [inline portality/formcontext/xwalk.py]: sure, though I'd expect every subject to have a code

**Comment by emanuil-tolev** [comment]
:+1: . Are you happy to release this? I could do too, or just be around, if you think it'll be troublesome. I guess the thing to watch out for is will the new CSV actually be good on live. deploy/crontab-production-apps tells me that the CSV runs every 30 min, so if you deploy around :50 you'd be able to check in about 10 minutes by going to /csv on the live site.

File: portality/formcontext/xwalk.py (hunk 1 above)
Comment (Steven-Eardley) [inline portality/formcontext/xwalk.py]: when I was demonstrating the CSV generation to @richard-jones he got an error on his local index and added this line to fix.

**Comment by Steven-Eardley** [comment]
I'll release this, thanks!

**Pull request merged**